theano/__init__.py

```diff
@@ -91,13 +91,15 @@
 
 # This need to be before the init of GPU, as it add config variable
 # needed during that phase.
-import theano.tests
-if hasattr(theano.tests, "TheanoNoseTester"):
-    test = theano.tests.TheanoNoseTester().test
-else:
-    def test():
-        raise ImportError("The nose module is not installed."
-                          " It is needed for Theano tests.")
+
+
+def test(*args):
```

Comment:
I used ```*args``` to return the results of the actual function, whenever this "fake" test function is called. Am I doing it in a right way?